runtime/src/main/java/org/corfudb/runtime/view/LayoutManagementView.java

```diff
@@ -162,7 +163,7 @@ public void addNode(Layout currentLayout,
             // TODO: Optimize this by retrying or submitting a workflow to retry.
             reconfigureSequencerServers(currentLayout, newLayout, true);
         } catch (InterruptedException | ExecutionException e) {
-            log.error("Bootstrapping sequencer failed due to exception : ", e);
+            throw new UnrecoverableCorfuInterruptedError("Bootstrapping sequencer failed", e);
```

Comment:
I don't think we should be throwing this for the ExecutionExceptions. It is supposed to fail silently in case of an exception. Else it will disrupt the add node workflow.
Throw this only for Interrupted Exceptions?

Reply:
Okay, done